operator/v1/types_machineconfiguration.go

```diff
@@ -54,6 +54,15 @@ type MachineConfigurationSpec struct {
 	// +openshift:enable:FeatureGate=NodeDisruptionPolicy
 	// +optional
 	NodeDisruptionPolicy NodeDisruptionPolicyConfig `json:"nodeDisruptionPolicy"`
+
+	// nodeSynchronizers is a list of node synchronizer that will be registered
+	// by this operator.  A node synchronizer manages a set of machines and
+	// ensures that they are updated to the latest configuration. The aggregate
+	// status of the machines managed by the node synchronizer will be reported
+	// in the Machine Configuration status.
+	// +openshift:enable:FeatureGate=PinnedImages
+	// +optional
+	NodeSynchronizers []NodeSynchronizerType `json:"nodeSynchronizers"`
```

Comment:
the spec field is new.  why a spec instead of just status?

Reply:
Still wondering about why we need the spec field.  The pinnedimageset thing is unconditional, right?